### PR TITLE
Add Finnish banks

### PR DESCRIPTION
**Pull request by jarmokortetjarvi**
Finnish banks and their basic information as a data import

### CODE REVIEW

File: l10n_fi_banks/README.rst [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+* Jarmo Kortetjärvi <jarmo.kortetjarvi@tawasta.fi>
+
+Maintainer
+----------
+
+.. image:: http://tawasta.fi/templates/tawastrap/images/logo.png
+   :alt: Oy Tawasta OS Technologies Ltd.
+   :target: http://tawasta.fi/
+
```
Comment (mlaitinen) [inline l10n_fi_banks/README.rst]: Shouldn't OCA be a maintainer as well? Like this:
```
.. image:: https://odoo-community.org/logo.png
   :alt: Odoo Community Association
   :target: https://odoo-community.org

This module is maintained by the OCA.

OCA, or the Odoo Community Association, is a nonprofit organization whose
mission is to support the collaborative development of Odoo features and
promote its widespread use.

To contribute to this module, please visit https://odoo-community.org.
```

File: l10n_fi_banks/data/res_bank.xml [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+        <!-- Finnish banks -->
+
+        <!-- TODO: Add main branch address information -->
+        <!-- TODO: Add branches? -->
+
+        <!-- Disable the Reserve-account -->
+        <!-- We could also delete this, but there is always a chance
+          someone modified this to be an actual account -->
+        <record id="base.res_bank_1" model="res.bank">
+            <field name="active" eval="0" />
```
Comment (mlaitinen) [inline l10n_fi_banks/data/res_bank.xml]: I think it's best to leave this out. It's a bit hazardous to deactivate a record if this module isn't installed in a fresh Odoo installation.

File: l10n_fi_banks/data/res_bank.xml [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+        </record>
+
+        <!-- Please maintain the alphabetical order when doing additions -->
```
Comment (mlaitinen) [inline l10n_fi_banks/data/res_bank.xml]: Consider adding the following banks as well.

- DnB NOR Bank ([DNBAFIHX](https://www.dnb.no/en/about-us/global-network/finland.html))
- Holvi ([HOLVFIHH](https://support.holvi.com/hc/en-gb/articles/201434362-Payments-to-Holvi-IBAN-and-BIC))

File: l10n_fi_banks/__init__.py [hunk 4]
```diff
@@ -0,0 +1 @@
+import models
```
Comment (mlaitinen) [inline l10n_fi_banks/__init__.py]: Maybe remove the `models` package and `__init__.py` inside of it?

File: l10n_fi_banks/__openerp__.py [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program. If not, see http://www.gnu.org/licenses/agpl.html
+#
+##############################################################################
+
+{
+    'name': 'Finnish Banks',
+    'summary': 'Finnish banks and their addresses',
+    'version': '8.0.0.4.0',
```
Comment (mlaitinen) [inline l10n_fi_banks/__openerp__.py]: `8.0.1.0.0` for the first "official" release? :)

**Review by mlaitinen — COMMENTED** [review]
Nice job

**Comment by jarmokortetjarvi** [comment]
I agree with the notes @mlaitinen gave, and implemented them.

File: l10n_fi_banks/__init__.py (hunk 4 above)
Comment (mlaitinen) [reply l10n_fi_banks/__init__.py]: At least one of the checks fail because of the `import models` still exists in the root `__init__.py`

Comment (dreispt) [reply l10n_fi_banks/__init__.py]: This import is breaking the build: see https://travis-ci.org/OCA/l10n-finland/jobs/227352221#L399

**Review by dreispt — APPROVED** [review]

**Review by mlaitinen — APPROVED** [review]

**Pull request merged**